schema.sql

```diff
@@ -24,4 +24,15 @@ ALTER TABLE animals DROP COLUMN species;
 
```

Comment:
**[OPTIONAL]**

- [ ] Kindly make your readme more descriptive by removing unnecessary details such as section descriptions, NOTES and details not related to this project 👍

Reply:
@HAMMAS-SALEEM Thank you for your feedback. However, It is not clear which part of my readme file is not required. I already removed all unnecessary sections that are not essential at this stage. I also don't understand why a short description of the project is not required. This has been accepted by almost all reviewers.